python_modules/dagster/dagster_tests/core_tests/storage_tests/test_local_instance.py

```diff
@@ -240,3 +241,19 @@ def should_not_execute(_, x):
         assert step_stats[0].end_time > step_stats[0].start_time
         assert step_stats[0].attempts == 4
         assert not _called
+
+
+def test_threaded_ephemeral_instance(caplog):
+    def _instantiate_ephemeral_instance():
+        with DagsterInstance.ephemeral():
+            pass
```

Comment:
if you actually use the instance and create a connection - do the errors start showing up again?